pom.xml

```diff
@@ -199,7 +199,7 @@
             <dependency>
                 <groupId>org.eclipse.che.dashboard</groupId>
                 <artifactId>che-dashboard-war</artifactId>
-                <version>${che.version}</version>
+                <version>5.18.0</version>
```

Comment:
I dont remember from daily call why we should use 5.18.0 UD ?

Reply:
That's because current dashboard v5.19 and higher has it's own keycloak client implementation and it doesn't make sense to patch it for rh-che assembly. That's why I've kept the old version. 

Another solution would have been to simply use upstream UD (without rebuilding it for rh-che anymore). But I am not sure that the upstream UD keycloak component would have worked on OSIO out of the box. @davidfestal what do you think?

Reply:
I'm not sure at all that this would work out-of-the-box.